.github/workflows/update.yml

```diff
@@ -13,18 +13,19 @@
 # limitations under the License.
 
 name: Run Data Pipeline
```

Comment:
Great question. A staging environment is something I'm hoping we can work on next. Perhaps we can bundle it as part of your testing work, or we can do it separately. Either way it will require some thinking and a design doc.

The auto-run is up and running on a GCE instance managed by Google.